docs/modules/ROOT/pages/index.adoc

```diff
@@ -11,6 +11,7 @@ include::./includes/attributes.adoc[]
 :vault-transit-guide: xref:vault-transit.adoc[Vault Transit Secret Engine Guide]
 :vault-pki-guide: xref:vault-pki.adoc[Vault PKI Secret Engine Guide]
 :vault-datasource-guide: xref:vault-datasource.adoc[Vault Datasource Guide]
+:vault-rabbitmq-guide: link:rabbitmq-reference#dynamic-credentials[RabbitMQ Dynamic Credentials Guide]
```

Comment:
where is this guide located?

Reply:
I didn't even think about this. This _was_ a link to the RabbitMQ reference doc in core docs. How do we link to guides/references in core now?

Reply:
/cc @gsmet

Reply:
https://quarkus.io/guides/rabbitmq-reference#dynamic-credentials

Reply:
Thanks! That's kind of what I figured.

Reply:
Fixed.